### ISSUE TITLE
Intellisense doesn't work in HTML without TS files open

### ISSUE THREAD

**Issue by dluk** [angular/vscode-ng-language-service#480]
Intellisense doesn't work

1. create new project using cli `ng new demo`
2. Click on 'app.component.html'
3. See error

1. Console output:
```
[Info  - 11:16:16 AM] Angular language server process ID: 3068
[Info  - 11:16:16 AM] Using typescript v3.6.4 from c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.0\node_modules\typescript\lib\tsserverlibrary.js
[Info  - 11:16:16 AM] Using @angular/language-service v9.0.0-rc.3 from c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.0\server\node_modules\@angular\language-service\bundles\language-service.umd.js
[Info  - 11:16:16 AM] Log file: c:\Users\dluk\AppData\Roaming\Code\logs\20191206T111609\exthost1\Angular.ng-template\nglangsvc.log
[Error - 11:16:19 AM] No config file for c:\Users\dluk\source\repos\dadada\demo\src\app\app.component.html

```
2. Log file:
```
Info 0    [11:16:16.190] Format host information updated
Info 1    [11:16:16.191] reload projects.
Info 2    [11:16:16.191] Structure before ensureProjectForOpenFiles:
Info 3    [11:16:16.191] Structure after ensureProjectForOpenFiles:
Info 4    [11:16:16.191] Host file extension mappings updated
Info 5    [11:16:16.200] Search path: c:/Users/dluk/source/repos/dadada/demo/src/app
Info 6    [11:16:16.201] For info: c:/Users/dluk/source/repos/dadada/demo/src/app/app.component.html :: Config file name: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 7    [11:16:16.201] Opened configuration file c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 8    [11:16:16.241] Loading global plugin @angular/language-service
Info 9    [11:16:16.241] Enabling plugin @angular/language-service from candidate paths: c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.0/node_modules/typescript/lib/tsserverlibrary.js/../../..,c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.0\server\node_modules\@angular\language-service\bundles\language-service.umd.js
Info 10   [11:16:16.241] Loading @angular/language-service from c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.0/node_modules/typescript/lib/tsserverlibrary.js/../../.. (resolved to c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.0/node_modules/node_modules)
Info 11   [11:16:16.326] Plugin activation warning: Missing proxied method getSmartSelectionRange in created LS. Patching.
Info 12   [11:16:16.326] Plugin activation warning: Missing proxied method getNonBoundSourceFile in created LS. Patching.
Info 13   [11:16:16.326] Plugin activation warning: Missing proxied method toLineColumnOffset in created LS. Patching.
Info 14   [11:16:16.326] Plugin activation warning: Missing proxied method getSourceMapper in created LS. Patching.
Info 15   [11:16:16.326] Plugin validation succeded
Info 16   [11:16:16.370] Starting updateGraphWorker: Project: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 17   [11:16:16.372] Finishing updateGraphWorker: Project: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json Version: 1 structureChanged: true Elapsed: 2ms
Info 18   [11:16:16.372] Project 'c:/Users/dluk/source/repos/dadada/demo/tsconfig.json' (Configured) 
Info 19   [11:16:16.372] 	Files (0)

Info 20   [11:16:16.372] -----------------------------------------------
Info 21   [11:16:16.373] Starting updateGraphWorker: Project: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 22   [11:16:18.489] Finishing updateGraphWorker: Project: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json Version: 2 structureChanged: true Elapsed: 2116ms
Info 23   [11:16:18.489] Project 'c:/Users/dluk/source/repos/dadada/demo/tsconfig.json' (Configured) 
Info 24   [11:16:18.489] 	Files (254)

Info 25   [11:16:18.489] -----------------------------------------------
Info 26   [11:16:18.493] Loading global plugin @angular/language-service
Info 27   [11:16:18.493] Enabling plugin @angular/language-service from candidate paths: c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.0/node_modules/typescript/lib/tsserverlibrary.js/../../..,c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.0\server\node_modules\@angular\language-service\bundles\language-service.umd.js
Info 28   [11:16:18.493] Loading @angular/language-service from c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.0/node_modules/typescript/lib/tsserverlibrary.js/../../.. (resolved to c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.0/node_modules/node_modules)
Info 29   [11:16:18.493] Plugin activation warning: Missing proxied method getSmartSelectionRange in created LS. Patching.
Info 30   [11:16:18.493] Plugin activation warning: Missing proxied method getNonBoundSourceFile in created LS. Patching.
Info 31   [11:16:18.493] Plugin activation warning: Missing proxied method toLineColumnOffset in created LS. Patching.
Info 32   [11:16:18.493] Plugin activation warning: Missing proxied method getSourceMapper in created LS. Patching.
Info 33   [11:16:18.493] Plugin validation succeded
Info 34   [11:16:18.496] Starting updateGraphWorker: Project: /dev/null/inferredProject1*
Info 35   [11:16:19.669] Finishing updateGraphWorker: Project: /dev/null/inferredProject1* Version: 1 structureChanged: true Elapsed: 1173ms
Info 36   [11:16:19.669] Project '/dev/null/inferredProject1*' (Inferred) 
Info 37   [11:16:19.669] 	Files (26)

Info 38   [11:16:19.669] -----------------------------------------------

```
extension version: `0.900.0`
package.json: 
```
{
  "name": "demo",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "~8.2.14",
    "@angular/common": "~8.2.14",
    "@angular/compiler": "~8.2.14",
    "@angular/core": "~8.2.14",
    "@angular/forms": "~8.2.14",
    "@angular/platform-browser": "~8.2.14",
    "@angular/platform-browser-dynamic": "~8.2.14",
    "@angular/router": "~8.2.14",
    "rxjs": "~6.4.0",
    "tslib": "^1.10.0",
    "zone.js": "~0.9.1"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.803.19",
    "@angular/cli": "~8.3.19",
    "@angular/compiler-cli": "~8.2.14",
    "@angular/language-service": "~8.2.14",
    "@types/node": "~8.9.4",
    "@types/jasmine": "~3.3.8",
    "@types/jasminewd2": "~2.0.3",
    "codelyzer": "^5.0.0",
    "jasmine-core": "~3.4.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.1.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~2.0.1",
    "karma-jasmine-html-reporter": "^1.4.0",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.15.0",
    "typescript": "~3.5.3"
  }
}

```

**Comment by kyliau** [comment]
This is a known problem with TypeScript 3.6
It'll be fixed once we upgrade to TypeScript 3.7, but we'll have to wait for Angular to upgrade first.

In the meantime, there's a simple solution: open a TS file first, then HTML, the problem will go away.

**Comment by dluk** [comment]
When I open a TS file first, there is no error anymore, but when I hit F12 on a bound property or method in a template it says `no definition found for`

**Comment by kyliau** [comment]
@dluk Could you provide the logs again to help us debug the issue? Please attach both the vscode output and the log file.

**Comment by dluk** [comment]
@kyliau vscode output:
```
[Info  - 8:52:45 AM] Angular language server process ID: 422036
[Info  - 8:52:45 AM] Using typescript v3.6.4 from c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.1\node_modules\typescript\lib\tsserverlibrary.js
[Info  - 8:52:45 AM] Using @angular/language-service v9.0.0-rc.5 from c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.1\server\node_modules\@angular\language-service\bundles\language-service.umd.js
[Info  - 8:52:45 AM] Log file: c:\Users\dluk\AppData\Roaming\Code\logs\20191210T085240\exthost1\Angular.ng-template\nglangsvc.log
[Error - 8:52:48 AM] No config file for c:\Users\dluk\source\repos\dadada\demo\src\app\app.component.html
```
log file: 
```
Info 0    [8:52:45.121] Format host information updated
Info 1    [8:52:45.122] reload projects.
Info 2    [8:52:45.123] Structure before ensureProjectForOpenFiles:
Info 3    [8:52:45.123] Structure after ensureProjectForOpenFiles:
Info 4    [8:52:45.123] Host file extension mappings updated
Info 5    [8:52:45.133] Search path: c:/Users/dluk/source/repos/dadada/demo/src/app
Info 6    [8:52:45.134] For info: c:/Users/dluk/source/repos/dadada/demo/src/app/app.component.html :: Config file name: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 7    [8:52:45.135] Opened configuration file c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 8    [8:52:45.169] Loading global plugin @angular/language-service
Info 9    [8:52:45.169] Enabling plugin @angular/language-service from candidate paths: c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/typescript/lib/tsserverlibrary.js/../../..,c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.1\server\node_modules\@angular\language-service\bundles\language-service.umd.js
Info 10   [8:52:45.169] Loading @angular/language-service from c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/typescript/lib/tsserverlibrary.js/../../.. (resolved to c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/node_modules)
Info 11   [8:52:45.242] Plugin activation warning: Missing proxied method getSmartSelectionRange in created LS. Patching.
Info 12   [8:52:45.242] Plugin activation warning: Missing proxied method getNonBoundSourceFile in created LS. Patching.
Info 13   [8:52:45.242] Plugin activation warning: Missing proxied method toLineColumnOffset in created LS. Patching.
Info 14   [8:52:45.242] Plugin activation warning: Missing proxied method getSourceMapper in created LS. Patching.
Info 15   [8:52:45.242] Plugin validation succeded
Info 16   [8:52:45.277] Starting updateGraphWorker: Project: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 17   [8:52:45.278] Finishing updateGraphWorker: Project: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json Version: 1 structureChanged: true Elapsed: 1ms
Info 18   [8:52:45.278] Project 'c:/Users/dluk/source/repos/dadada/demo/tsconfig.json' (Configured) 
Info 19   [8:52:45.278] 	Files (0)

Info 20   [8:52:45.278] -----------------------------------------------
Info 21   [8:52:45.279] Starting updateGraphWorker: Project: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 22   [8:52:47.210] Finishing updateGraphWorker: Project: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json Version: 2 structureChanged: true Elapsed: 1931ms
Info 23   [8:52:47.210] Project 'c:/Users/dluk/source/repos/dadada/demo/tsconfig.json' (Configured) 
Info 24   [8:52:47.210] 	Files (254)

Info 25   [8:52:47.210] -----------------------------------------------
Info 26   [8:52:47.214] Loading global plugin @angular/language-service
Info 27   [8:52:47.214] Enabling plugin @angular/language-service from candidate paths: c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/typescript/lib/tsserverlibrary.js/../../..,c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.1\server\node_modules\@angular\language-service\bundles\language-service.umd.js
Info 28   [8:52:47.214] Loading @angular/language-service from c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/typescript/lib/tsserverlibrary.js/../../.. (resolved to c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/node_modules)
Info 29   [8:52:47.214] Plugin activation warning: Missing proxied method getSmartSelectionRange in created LS. Patching.
Info 30   [8:52:47.214] Plugin activation warning: Missing proxied method getNonBoundSourceFile in created LS. Patching.
Info 31   [8:52:47.214] Plugin activation warning: Missing proxied method toLineColumnOffset in created LS. Patching.
Info 32   [8:52:47.214] Plugin activation warning: Missing proxied method getSourceMapper in created LS. Patching.
Info 33   [8:52:47.214] Plugin validation succeded
Info 34   [8:52:47.217] Starting updateGraphWorker: Project: /dev/null/inferredProject1*
Info 35   [8:52:48.231] Finishing updateGraphWorker: Project: /dev/null/inferredProject1* Version: 1 structureChanged: true Elapsed: 1014ms
Info 36   [8:52:48.231] Project '/dev/null/inferredProject1*' (Inferred) 
Info 37   [8:52:48.231] 	Files (26)

Info 38   [8:52:48.231] -----------------------------------------------
Info 39   [8:52:50.948] Search path: c:/Users/dluk/source/repos/dadada/demo/src/app
Info 40   [8:52:50.948] For info: c:/Users/dluk/source/repos/dadada/demo/src/app/app.component.ts :: Config file name: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 41   [8:52:50.949] `remove Project::
Info 42   [8:52:50.949] Project '/dev/null/inferredProject1*' (Inferred) 
Info 43   [8:52:50.949] 	Files (26)

Info 44   [8:52:50.949] -----------------------------------------------
Info 45   [8:52:51.181] Computing Angular semantic diagnostics...
Info 46   [8:52:57.623] Search path: c:/Users/dluk/source/repos/dadada/demo/src/app
Info 47   [8:52:57.623] For info: c:/Users/dluk/source/repos/dadada/demo/src/app/app.component.html :: Config file name: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 48   [8:52:57.624] Loading global plugin @angular/language-service
Info 49   [8:52:57.624] Enabling plugin @angular/language-service from candidate paths: c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/typescript/lib/tsserverlibrary.js/../../..,c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.1\server\node_modules\@angular\language-service\bundles\language-service.umd.js
Info 50   [8:52:57.624] Loading @angular/language-service from c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/typescript/lib/tsserverlibrary.js/../../.. (resolved to c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/node_modules)
Info 51   [8:52:57.624] Plugin activation warning: Missing proxied method getSmartSelectionRange in created LS. Patching.
Info 52   [8:52:57.624] Plugin activation warning: Missing proxied method getNonBoundSourceFile in created LS. Patching.
Info 53   [8:52:57.624] Plugin activation warning: Missing proxied method toLineColumnOffset in created LS. Patching.
Info 54   [8:52:57.624] Plugin activation warning: Missing proxied method getSourceMapper in created LS. Patching.
Info 55   [8:52:57.624] Plugin validation succeded
Info 56   [8:52:57.628] Starting updateGraphWorker: Project: /dev/null/inferredProject2*
Info 57   [8:52:58.254] Finishing updateGraphWorker: Project: /dev/null/inferredProject2* Version: 1 structureChanged: true Elapsed: 626ms
Info 58   [8:52:58.254] Project '/dev/null/inferredProject2*' (Inferred) 
Info 59   [8:52:58.254] 	Files (26)

Info 60   [8:52:58.254] -----------------------------------------------
Info 61   [8:52:58.455] Search path: c:/Users/dluk/source/repos/dadada/demo/src/app
Info 62   [8:52:58.455] For info: c:/Users/dluk/source/repos/dadada/demo/src/app/app.component.html :: Config file name: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 63   [8:52:58.456] Computing Angular semantic diagnostics...

```

**Comment by dluk** [comment]
Another one where I opened TS file first, vscode output:
```
[Info  - 8:55:57 AM] Angular language server process ID: 420768
[Info  - 8:55:57 AM] Using typescript v3.6.4 from c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.1\node_modules\typescript\lib\tsserverlibrary.js
[Info  - 8:55:57 AM] Using @angular/language-service v9.0.0-rc.5 from c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.1\server\node_modules\@angular\language-service\bundles\language-service.umd.js
[Info  - 8:55:57 AM] Log file: c:\Users\dluk\AppData\Roaming\Code\logs\20191210T085553\exthost1\Angular.ng-template\nglangsvc.log
```
log file: 
```
Info 0    [8:55:57.140] Format host information updated
Info 1    [8:55:57.141] reload projects.
Info 2    [8:55:57.141] Structure before ensureProjectForOpenFiles:
Info 3    [8:55:57.141] Structure after ensureProjectForOpenFiles:
Info 4    [8:55:57.141] Host file extension mappings updated
Info 5    [8:55:57.154] Search path: c:/Users/dluk/source/repos/dadada/demo/src/app
Info 6    [8:55:57.155] For info: c:/Users/dluk/source/repos/dadada/demo/src/app/app.component.ts :: Config file name: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 7    [8:55:57.156] Opened configuration file c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 8    [8:55:57.211] Loading global plugin @angular/language-service
Info 9    [8:55:57.211] Enabling plugin @angular/language-service from candidate paths: c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/typescript/lib/tsserverlibrary.js/../../..,c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.1\server\node_modules\@angular\language-service\bundles\language-service.umd.js
Info 10   [8:55:57.212] Loading @angular/language-service from c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/typescript/lib/tsserverlibrary.js/../../.. (resolved to c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/node_modules)
Info 11   [8:55:57.319] Plugin activation warning: Missing proxied method getSmartSelectionRange in created LS. Patching.
Info 12   [8:55:57.319] Plugin activation warning: Missing proxied method getNonBoundSourceFile in created LS. Patching.
Info 13   [8:55:57.319] Plugin activation warning: Missing proxied method toLineColumnOffset in created LS. Patching.
Info 14   [8:55:57.319] Plugin activation warning: Missing proxied method getSourceMapper in created LS. Patching.
Info 15   [8:55:57.319] Plugin validation succeded
Info 16   [8:55:57.374] Starting updateGraphWorker: Project: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 17   [8:55:57.376] Finishing updateGraphWorker: Project: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json Version: 1 structureChanged: true Elapsed: 2ms
Info 18   [8:55:57.377] Project 'c:/Users/dluk/source/repos/dadada/demo/tsconfig.json' (Configured) 
Info 19   [8:55:57.377] 	Files (0)

Info 20   [8:55:57.377] -----------------------------------------------
Info 21   [8:55:57.378] Starting updateGraphWorker: Project: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 22   [8:56:0.152] Finishing updateGraphWorker: Project: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json Version: 2 structureChanged: true Elapsed: 2774ms
Info 23   [8:56:0.152] Project 'c:/Users/dluk/source/repos/dadada/demo/tsconfig.json' (Configured) 
Info 24   [8:56:0.152] 	Files (254)

Info 25   [8:56:0.152] -----------------------------------------------
Info 26   [8:56:0.389] Computing Angular semantic diagnostics...
Info 27   [8:56:2.557] Search path: c:/Users/dluk/source/repos/dadada/demo/src/app
Info 28   [8:56:2.558] For info: c:/Users/dluk/source/repos/dadada/demo/src/app/app.component.html :: Config file name: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 29   [8:56:2.558] Loading global plugin @angular/language-service
Info 30   [8:56:2.559] Enabling plugin @angular/language-service from candidate paths: c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/typescript/lib/tsserverlibrary.js/../../..,c:\Users\dluk\.vscode\extensions\angular.ng-template-0.900.1\server\node_modules\@angular\language-service\bundles\language-service.umd.js
Info 31   [8:56:2.559] Loading @angular/language-service from c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/typescript/lib/tsserverlibrary.js/../../.. (resolved to c:/Users/dluk/.vscode/extensions/angular.ng-template-0.900.1/node_modules/node_modules)
Info 32   [8:56:2.559] Plugin activation warning: Missing proxied method getSmartSelectionRange in created LS. Patching.
Info 33   [8:56:2.559] Plugin activation warning: Missing proxied method getNonBoundSourceFile in created LS. Patching.
Info 34   [8:56:2.559] Plugin activation warning: Missing proxied method toLineColumnOffset in created LS. Patching.
Info 35   [8:56:2.559] Plugin activation warning: Missing proxied method getSourceMapper in created LS. Patching.
Info 36   [8:56:2.559] Plugin validation succeded
Info 37   [8:56:2.562] Starting updateGraphWorker: Project: /dev/null/inferredProject1*
Info 38   [8:56:3.568] Finishing updateGraphWorker: Project: /dev/null/inferredProject1* Version: 1 structureChanged: true Elapsed: 1006ms
Info 39   [8:56:3.568] Project '/dev/null/inferredProject1*' (Inferred) 
Info 40   [8:56:3.568] 	Files (26)

Info 41   [8:56:3.568] -----------------------------------------------
Info 42   [8:56:3.635] Search path: c:/Users/dluk/source/repos/dadada/demo/src/app
Info 43   [8:56:3.635] For info: c:/Users/dluk/source/repos/dadada/demo/src/app/app.component.html :: Config file name: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 44   [8:56:3.769] Computing Angular semantic diagnostics...
Info 45   [8:56:4.274] Search path: c:/Users/dluk/source/repos/dadada/demo/src/app
Info 46   [8:56:4.274] For info: c:/Users/dluk/source/repos/dadada/demo/src/app/app.component.ts :: Config file name: c:/Users/dluk/source/repos/dadada/demo/tsconfig.json
Info 47   [8:56:4.274] `remove Project::
Info 48   [8:56:4.274] Project '/dev/null/inferredProject1*' (Inferred) 
Info 49   [8:56:4.274] 	Files (26)

Info 50   [8:56:4.274] -----------------------------------------------
Info 51   [8:56:4.477] Computing Angular semantic diagnostics...

```

**Comment by kyliau** [comment]
@dluk It looks like the TS project was immediately removed once the TS file was closed. 
I should clarify that you should **keep the TS file open** while opening a HTML file, this would prevent the project from going away.
Again, this'll be solved entirely once we upgrade to TS 3.7

**Comment by ayazhafiz** [comment]
This was fixed by #551 and should be available in the next release.
